client/Brocfile.js

```diff
@@ -1,11 +1,23 @@
-/* global require, module */
-
+/* global require, module, process */
+var env = process.env.EMBER_ENV;
```

Comment:
The method suggested by  [ember-cli-google-analytics] (https://github.com/pgrippi/ember-cli-google-analytics#configuration) seems more elegant to me.  Not that we need to use that addon, but the idea of adding a property to ENV seems better than a environment var set in the OS.